src/js/components/SearchResults.js

```diff
@@ -27,11 +27,13 @@ const SearchResults = ({
   const encodedTerm = encodeURIComponent(term);
   let results = [...rawResults];
 
-  results = sortItems(
-    results,
-    (type === 'tracks' && sortField === 'followers' ? 'popularity' : sortField),
-    sortReverse,
-  );
+  if (sortField !== 'asloaded') {

```

Comment:
"As loaded" is `null`, so we would invert the logic here to `if (sortField) { ... then sort `